danielssonn/ethTO
# feat: implement swing ui

## Purpose

The primary purpose of this PR was to wire up the Swing UI with the Swing business logic, but I also made these small changes:

1. Renamed the `Rent` component to `Arrivals` and the `RentCheckout` component to `Checkout`
2. Fixed footer links
3. Added missing `alt` properties and updated `alt` values

## Testing Instructions

1. Confirm the Swing flow looks correct (optional):

- Pull down the branch
- Move into the `client` directory and run `npm run dev`
- Navigate to `http://localhost:3000/checkout/0x123b30e25973fecd8354dd5f41cc45a3065ef88c/1` (this is just a random contract)
- Click on Continue

2. Review code

## Checklist
